### PR TITLE
cloud: add table filters and dumpling docs

### PR DESCRIPTION
**Pull request by qiancai**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v6.2 (TiDB 6.2 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/import-parquet-files.md [hunk 1]
```diff
@@ -85,7 +85,7 @@ To import the Parquet files to TiDB Cloud, take the following steps:
     - **Data Format**: select **Parquet**.
     - **Setup Credentials** (This field is visible only for AWS S3): enter the Role ARN value for **Role-ARN**.
     - **Target Database**: fill in the **Username** and **Password** fields.
-    - **DB/Tables Filter**: if necessary, you can specify a [table filter](https://docs.pingcap.com/tidb/stable/table-filter#cli). Currently, TiDB Cloud only supports one table filter rule.
+    - **DB/Tables Filter**: if necessary, you can specify a [table filter](/table-filter.md#syntax). Currently, TiDB Cloud only supports one table filter rule.
```
Comment (Frank945946) [inline tidb-cloud/import-parquet-files.md:88]: Now ，user can use "," to separate multiple filter rules.

File: tidb-cloud/import-csv-files.md [hunk 2]
```diff
@@ -82,7 +82,7 @@ To import the CSV files to TiDB Cloud, take the following steps:
         > For the configurations of separator, delimiter, and null, you can use both alphanumeric characters and certain special characters. The supported special characters include `\t`, `\b`, `\n`, `\r`, `\f`, and `\u0001`.
 
     - **Target Database**: fill in the **Username** and **Password** fields.
-    - **DB/Tables Filter**: if necessary, you can specify a [table filter](https://docs.pingcap.com/tidb/stable/table-filter#cli). Currently, TiDB Cloud only supports one table filter rule.
+    - **DB/Tables Filter**: if necessary, you can specify a [table filter](/table-filter.md#syntax). Currently, TiDB Cloud only supports one table filter rule.
```
Comment (Frank945946) [inline tidb-cloud/import-csv-files.md:85]: Now ，user can use "," to separate multiple filter rules.

**Review by Frank945946 — COMMENTED** [review]
After 2022-07-26，users can use "," to separate multiple table filter rules

File: tidb-cloud/migrate-data-into-tidb.md [hunk 3]
```diff
@@ -7,7 +7,7 @@ summary: Learn how to migrate data from MySQL-compatible databases to TiDB Cloud
 
 TiDB is highly compatible with MySQL. You can migrate data from any MySQL-compatible databases to TiDB smoothly, whether the data is from a self-hosted MySQL instance or RDS service provided by the public cloud.
 
-This document describes how to use [Dumpling](https://docs.pingcap.com/tidb/stable/dumpling-overview) to export data from MySQL-compatible databases and use [TiDB Lightning](https://docs.pingcap.com/tidb/stable/tidb-lightning-overview) TiDB-backend to import the data to TiDB Cloud. 
+This document describes how to use [Dumpling](/dumpling-overview.md) to export data from MySQL-compatible databases and use [TiDB Lightning](https://docs.pingcap.com/tidb/stable/tidb-lightning-overview) TiDB-backend to import the data to TiDB Cloud.
```
Comment (ran-huang) [inline tidb-cloud/migrate-data-into-tidb.md:10]: TiDB-backend is a deprecated term. Do you want to change it to "Logical import mode"?

**Review by ran-huang — APPROVED** [review]
Rest LGTM

**Comment by qiancai** [comment]
@Frank945946 @ran-huang  Many thanks for your review. I've checked and updated the docs according to your comments. PTAL.

**Review by Frank945946 — APPROVED** [review]
LGTM

**Review by ran-huang — APPROVED** [review]

**Pull request merged**